web-sourcecode/src/components/SumaClient.vue

```diff
@@ -254,6 +257,21 @@ export default {
         }
       });
     },
+    loadLastInit: function(){
```

Comment:
This seems like an unnecessary additional function. I think 'currentinit' should get added to the list in loadCounts. I also think the function should get renamed since it no longer is just loading counts.